.github/actions/pnpm-setup/action.yml

```diff
@@ -0,0 +1,31 @@
+name: Install & setup pnpm
+description: Install's node, pnpm, restores cache
+
+runs:
+  using: 'composite'
+  steps:
+    # Install nodejs. https://github.com/actions/setup-node
+    - name: Setup Node.js
+      uses: actions/setup-node@v3
+      with:
+        node-version: 16.x
```

Comment:
Should we upgrade to 18 here?

Reply:
I'm not sure if there might be somone on node 16. It's could be quite dramatic in theory. SST seems to use 16 in their workflows. "You'll need at least [Node.js 16.6](https://nodejs.org/) and [npm 7](https://www.npmjs.com/). You also need to have an AWS account and [AWS credentials configured locally](https://docs.sst.dev/advanced/iam-credentials#loading-from-a-file)." from SST docs

Reply:
Ok, let's just leave for now. Hopefully pnpm 8 doesn't complain